docs/snapshots.asciidoc

```diff
@@ -0,0 +1,190 @@
+[id="{p}-how-to-snapshot"]
+= How to create automated snapshots
+
+To set up automated snapshots for Elasticsearch on Kubernetes you have to:
+
+. Ensure you have the necessary Elasticsearch storage plugin installed.
+. Add snapshot repository credentials to the Elasticsearch keystore.
+. Register the snapshot repository with the Elasticsearch API.
+. Set up a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob] to take snapshots on a schedule.
+
+The examples below use the https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-gcs.html[Google Cloud Storage Repository Plugin].
+
+For more information on Elasticsearch snapshots, see https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html[Snapshot and Restore].
+
+[id="{p}-install-plugin"]
+== Install the Storage Repository Plugin
+
+You can install the storage repository plugin either by using a custom image or by adding your own init container which
+installs the plugin at pod creation time.
+
```

Comment:
=> suggested rewording:

`To install the storage repository plugin, you can either use a custom image or add your own init container which installs the plugin at pod creation time.`